src/srpe.hpp

```diff
@@ -0,0 +1,40 @@
+#ifndef SRPE_x
+#define SRPE_x
+#include "filter.hpp"
+#include "index.hpp"
+#include "vg.pb.h"
+#include "vg.hpp"
+#include "gcsa.h"
+#include "alignment.hpp"
+#include "mapper.hpp"
+
+/**
```

Comment:
This needs to move inside the namespace, or the doc generator will think it's documentation for the vg namespace itself.